website/content/api-docs/secret/transform.mdx

```diff
@@ -356,6 +356,208 @@ $ curl \
+provided AES key. The wrapped AES key should be the first 512 bytes of the
+ciphertext, and the encrypted key material should be the remaining bytes.
+
+- `hash_function` `(string: "SHA256")` - The hash function used for the
+RSA-OAEP step of creating the ciphertext. Supported hash functions are:
+`SHA1`, `SHA224`, `SHA256`, `SHA384`, and `SHA512`. If not specified,
+the hash function defaults to SHA256.
+
+- `template` `(string: <required>)` -
+  Specifies the template name to use for matching value on encode and decode
+  operations when using this transformation.
+
+- `tweak_source` `(string: "supplied")` -
+  Specifies the source of where the tweak value comes from. Valid sources are
+  `supplied`, `generated`, and `internal`. Only used when the type is FPE.
+
+- `allowed_roles` `(list: [])` -
+  Specifies a list of allowed roles that this transformation can be assigned to.
+  A role using this transformation must exist in this list in order for
+  encode and decode operations to properly function.
+
+### Sample Payload
+
+```json
+{
+  "ciphertext": "...",
+  "template": "builtin/creditcardnumber",
+  "tweak_source": "internal",
+  "allowed_roles": ["example-role"]
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    https://127.0.0.1:8200/v1/transform/transformations/fpe/example-transformation/import
+```
+
+## Import Key for Tokenization Transformation
+
+This endpoint imports an existing AES-256 key into a new tokenization transformation.
+To import key material into an existing key, see the tokenization `import_version/` endpoint.
+
+| Method | Path                                                   |
+| :----- | :----------------------------------------------------- |
+| `POST` | `/transform/transformations/tokenization/:name/import` |
+
+### Parameters
+
+- `name` `(string: <required>)` – Specifies the name of the encryption key to
+  create. This is specified as part of the URL.
+
+- `ciphertext` `(string: <required>)` - A base64-encoded string that contains
+two values: an ephemeral 256-bit AES key wrapped using the wrapping key
+returned by Vault and the encryption of the import key material under the
+provided AES key. The wrapped AES key should be the first 512 bytes of the
+ciphertext, and the encrypted key material should be the remaining bytes.
+
+- `hash_function` `(string: "SHA256")` - The hash function used for the
+RSA-OAEP step of creating the ciphertext. Supported hash functions are:
+`SHA1`, `SHA224`, `SHA256`, `SHA384`, and `SHA512`. If not specified,
+the hash function defaults to SHA256.
+
+- `allow_rotation` `(bool: false)` - If set, the imported key can be rotated
+within Vault by using the `rotate` endpoint.
+
+~> **NOTE**: Once an imported key is rotated within Vault, it will no longer
+support importing key material with the `import_version` endpoint.
+
+- `mapping_mode` `(string: "default")` -
```

Comment:
Did we lose the "convergent" parameter?

Reply:
We dropped it from both Transit and Transform BYOK (at least for the time being) due to lack of attached HMAC key import support.